wls-broadcast-service/src/main/java/de/muenchen/oss/wahllokalsystem/broadcastservice/configuration/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,45 @@
+package de.muenchen.oss.wahllokalsystem.broadcastservice.configuration;
+
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.TechnischeWlsException;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.WlsException;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.errorhandler.AbstractExceptionHandler;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.rest.model.DTOMapper;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.rest.model.WlsExceptionDTO;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.servlet.resource.NoResourceFoundException;
+
+@Slf4j
+@ControllerAdvice
+public class GlobalExceptionHandler extends AbstractExceptionHandler {
+
+    @Value("${service.info.oid}")
+    String serviceOid;
+
+    public GlobalExceptionHandler(DTOMapper mapper) {
+        super(mapper);
+    }
+
+    @ResponseStatus(value = HttpStatus.NOT_FOUND)
+    @ExceptionHandler(NoResourceFoundException.class)
+    public WlsException handleException(Exception throwable) {
+        WlsException wlsException;
+        try {
+            WlsExceptionDTO wlsExceptionDTO = getWahlExceptionDTO(throwable);
+            wlsException = TechnischeWlsException.withCode(wlsExceptionDTO.code()).inService(getService()).withCause(throwable)
+                    .buildWithMessage(wlsExceptionDTO.message());
+        } catch (WlsException e) {
+            wlsException = e;
+        }
+        return wlsException;
+    }
```

Comment:
Sorgt dafür dass Exception aus der Exception ein DTO wird und ResponseStatus gesetzt wird (sofern) es ein wls-common:exception ExceptionKonstanten-Code ist.

Auf jeden fall muss die Methode rein um andere Fehler sauber nach außen zu mappen. Ein `POST` auf `http://localhost:39146/businessActions/messageRead` mit dem Body `{}` wird in der aktuellen Implementierung nicht sauber gehandelt. Die Response eine HTML-Seite und kein `WlsExceptionDTO`.

```suggestion
    @ExceptionHandler
    @ResponseBody
    public ResponseEntity<WlsExceptionDTO> handleException(Exception throwable) {
        return createResponse(getWahlExceptionDTO(throwable));
    }
```

Reply:
d31222ba078ef64526fc6fdf1764e5b43743ebba